### DISCUSSION TITLE
How to scan all images for updates?

### DISCUSSION THREAD

**Discussion by schklom** [containrrr/watchtower#1136] [Q&A] [answered]
My setup is to have Watchtower run once daily on Docker with `WATCHTOWER_RUN_ONCE: "true"` set. This lets me avoid running it constantly and waste CPU.

The problem is some updates haven't been pulled in months, despite being available on DockerHub.\
Last run ended with `Session done: 9 scanned, 1 updated, 0 failed` but I was able to manually pull new images for other containers hours later (I have about 30 running), so Watchtower only checks some images per session for some undocumented reason.

How can I force check all images?

Many thanks for all answers :)

**Comment by schklom** [comment, marked as answer]
I just realized Watchtower starts while others are starting, and so doesn't update every container.

**Comment by simonaronsson** [comment]
> This lets me avoid running it constantly and waste CPU.

If you have it set to scan at a daily interval, it won't waste any noticeable amount of CPU in between runs [as it's basically dormant](https://github.com/containrrr/watchtower/blob/6ff1d8ac17e4244bb7c1adb41ba41e9b370c933b/cmd/root.go#L322-L341).
